### ISSUE TITLE
Is it possible to use grouped and stacked bar in one chart

### ISSUE THREAD

**Issue by asdasd293** [rrag/react-stockcharts#808]
![CS_Col_00](https://user-images.githubusercontent.com/28852948/163978135-b045cf53-e0b1-4bcf-a1ab-98aa2c5e619b.png)
I want to create a chart like this. Is it possible?

**Comment by Devamchaudhari** [comment]
@asdasd293 Yes you can, http://rrag.github.io/react-stockcharts/documentation.html#/stacked_bar_chart
